apps/src/p5lab/utils.js

```diff
@@ -0,0 +1,6 @@
+// NOTE: min and max are inclusive
+export const utils = {
+  randomInt(min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+};
```

Comment:
we generally use a similar but slightly different paradigm for exporting a set of helper functions from a file. i just created this file so i'll use it as an example--

here, i'm exporting the `getTextWidth` helper function:
https://github.com/code-dot-org/code-dot-org/blob/c014b6521d9712a12f4a81c1db4905c50eefff13/apps/src/p5lab/drawUtils.js#L13-L20

and when i import it, i can reference the drawUtils.js helpers as a single object even though they're all exported separately:

https://github.com/code-dot-org/code-dot-org/blob/c014b6521d9712a12f4a81c1db4905c50eefff13/apps/src/p5lab/spritelab/CoreLibrary.js#L2

and it gets called basically exactly the same as how your helper is called now:
https://github.com/code-dot-org/code-dot-org/blob/c014b6521d9712a12f4a81c1db4905c50eefff13/apps/src/p5lab/spritelab/CoreLibrary.js#L99

this pattern is basically the same as what you have, but simplifies some nesting and is basically syntactic sugar. interestingly, you _could_ import just the function you want like this (still using my example):
```javascript
import {getTextWidth} from '@cdo/apps/p5lab/drawUtils'; 
```

however, that approach can make unit testing more difficult (which we can talk about sometime).

those are the basics, but MDN has pretty good articles about [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [export](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) as well!